extensions/test-extension/src/getCustomizationModule.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  codingValues,
+  contextMenuCodeItem,
+  findingsContextMenu,
+} from './custom-context-menu';
+
+export default function getCustomizationModule() {
+  return [
+    {
+      name: 'custom-context-menu',
+      value: [codingValues, contextMenuCodeItem, findingsContextMenu],
```

Comment:
how is `codingValues` passed here, it is just an object, the rest (contexxtMenuCodeItem and findingsContextMenu) on the other hand follow the interface for customization

Reply:
All values passed into the customization module are just objects.    This, and every other instance passed in gets used by someone using the customization service, but the only default/handling is that customizationType gets inheritted from, and transform gets called to update the object, but neither of those are used here.